docs/source/API/alphabetical.rst

```diff
@@ -0,0 +1,231 @@
+| `resize <core/view/resize.html>`_                                                    | `Core <core-index.html>`_ | `View and related <core/View.html>`_                                   | Resize an existing view while maintaining the content                                                                                   |
++--------------------------------------------------------------------------------------+---------------------------+------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
+| `Serial <core/execution_spaces.html#kokkos-serial>`_                                 | `Core <core-index.html>`_ | `Spaces <core/Spaces.html>`_                                           | Execution space using serial execution the CPU.                                                                                         |
++--------------------------------------------------------------------------------------+---------------------------+------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
+| `ScopeGuard <core/initialize_finalize/ScopeGuard.html>`_                             | `Core <core-index.html>`_ | `Initialization and Finalization <core/Initialize-and-Finalize.html>`_ | class to aggregate initializing and finalizing Kokkos                                                                                   |
++--------------------------------------------------------------------------------------+---------------------------+------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
+| `SpaceAccessibility <core/SpaceAccessibility.html>`_                                 | `Core <core-index.html>`_ | `Spaces <core/Spaces.html>`_                                           | Facility to query accessibility rules between execution and memory spaces.                                                              |
++--------------------------------------------------------------------------------------+---------------------------+------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------+
+| `Subview Type <core/view/Subview_type.html>`_                                        | `Core <core-index.html>`_ | `View and related <core/View.html>`_                                   | Type of multi-dimensional array which is returned by the subview function                                                               |
```

Comment:
We can't do this: the keyword is `Subview` as stupid as it is.

Reply:
looking into how to get rid of "Type" but avoid warning